books/microbiology/bake

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '7.0.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :microbiology) do |doc|
+  include Kitchen::Directions
+
+  book = doc.book
+  metadata = book.metadata
+
+  # Some stuff just goes away
+  book.search('cnx-pi').trash
+
+  BakePreface.v1(book: book, title_element: 'h1')
+  BakeChapterIntroductions.v1(book: book)
+  BakeChapterTitle.v1(book: book)
+  book.chapters.each do |chapter|
+    BakePageAbstracts.v1(chapter: chapter)
+    BakeNonIntroductionPages.v1(chapter: chapter)
+  end
+
+  BakeAutotitledNotes.v1(
+    book: book,
+    classes: %w[clinical-focus check-your-understanding micro-connection
+                link-to-learning eye-on-ethics case-in-point disease-profile]
+  )
+
+  book_answer_key = BookAnswerKeyContainer.v1(book: book, solutions_plural: false)
+  book.chapters.each do |chapter|
+    BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata)
+
+    eoc_wrapper = ChapterReviewContainer.v1(chapter: chapter, metadata_source: metadata, klass: 'review-questions')
+
+    exercise_section_classes = %w[multiple-choice true-false matching
+                                  fill-in-the-blank short-answer critical-thinking]
+
+    exercise_section_classes.each do |klass|
+      MoveExercisesToEOC.v1(chapter: chapter, metadata_source: metadata, append_to: eoc_wrapper, klass: klass)
+    end
+
+    chapter.search(exercise_section_classes.map{ |klass| "section.#{klass}" }.join(', ')).exercises
+           .each do |exercise|
+      BakeNumberedExercise.v1(exercise: exercise, number: exercise.count_in(:chapter))
+    end
+
+    # Bake answer key from chapter/ move solutions from eoc into answer key
+    MoveSolutionsToAnswerKey.v1(
+      chapter: chapter,
+      metadata_source: metadata,
+      strategy: :default,
+      append_to: book_answer_key,
+      strategy_options: {
+        selectors: %w[
+          multiple-choice
+          true-false matching
```

Comment:
is it intended that `critical-thinking` isn't here?  if it should be here, use `exercise_section_classes.prefix('section.) instead.  

If it shouldn't be here, maybe use `exercise_section_classes.except('critical-thinking')`?

If want to leave it as is, please put `matching` on its own line.